examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -12,50 +12,47 @@ import {
 	typeboxValidator,
 } from "@fluid-experimental/tree2";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
-import { InventoryField, inventoryField, schema } from "./schema";
+import { InventoryField, schema } from "./schema";
 
 const treeKey = "tree";
 
 const factory = new TypedTreeFactory({
 	jsonValidator: typeboxValidator,
 	forest: ForestType.Reference,
-	initialTree: {
-		parts: [
-			{
-				name: "nut",
-				quantity: 0,
-			},
-			{
-				name: "bolt",
-				quantity: 0,
-			},
-		],
-	},
-	allowedSchemaModifications: AllowedUpdateType.None,
-	schema,
 	subtype: "InventoryList",
 });
 
 export class InventoryList extends DataObject {
-	private _tree: TypedTreeChannel<typeof inventoryField> | undefined;
+	private _tree: TypedTreeChannel | undefined;
 
 	public get tree(): InventoryField {
 		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		return this._tree!.root;
+		return this._tree!.schematize({
```

Comment:
Why is it safe to assume `_tree` is defined? Can we document that above the lint disable?